src/ContentType/ContentType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of sensiolabs-de/storyblok-bundle.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Bundle\ContentType;
+
+use function Symfony\Component\String\u;
+
+abstract readonly class ContentType implements ContentTypeInterface
+{
+    final public static function type(): string
```

Comment:
The logic we use here might fit our use case, but what if someone has more folders for their namespace?

TLDR: I would allow to overwrite this here, let's be more open, instead to force the user to use it this way, WDYT?

Reply:
Good catch!

Reply:
> The logic we use here might fit our use case, but what if someone has more folders for their namespace?

That should not matter because it converts the current class name to the content_type name which you define in storyblok such as `page` or `blog_post`. Due to storybloks API specs they use snake_case for the keys so we should use them too but i totally agree with you we should make it overridable